agent/invocation/recipes.rb

```diff
@@ -18,20 +18,20 @@ def setup_repo(git_url)
       Dir.chdir(dir)
       logger.debug("Changing into working directory #{dir}.")
 
+      # TOOD: need Git Credentials for private repos.
       sh("git clone --depth 1 #{git_url} repo")
+      # sh("git checkout <get_the_sha_from_env>")
```

Comment:
Checking out the specific sha is a requirement to get the PR merged 👍

Reply:
Done 👍